### DISCUSSION TITLE
Performance metrics does not work after upgrading to 5.3.7

### DISCUSSION THREAD

**Discussion by mlcrm** [symfony/symfony#42865] [Q&A] [answered]
After upgrading Symfony version to 5.3.7, Performance metrics doesn't work. Execution timeline - ["No timing events have been recorded. Check that symfony/stopwatch is installed and debugging enabled in the kernel"]. Is this a bug? When downgrading to version 5.3.6 everything works fine

**Comment by fabpot** [comment, marked as answer]
It is a bug that has been fixed now. It will be part of the next release.

**Reply by stof** [reply]
@fabpot maybe we should do the release, as this has already been reported multiple times ?
